spec/vcr/advocate_types_spec.yml

```diff
@@ -2,130 +2,121 @@
 http_interactions:
 - request:
     method: get
-    uri: https://laa-fee-calculator.service.justice.gov.uk/api/v1/fee-schemes/1/
+    uri: https://laa-fee-calculator.service.justice.gov.uk/api/v1/fee-schemes/1/advocate-types/
     body:
       encoding: US-ASCII
       string: ''
     headers:
       Accept:
       - application/json
       User-Agent:
-      - laa-fee-calculator-client/0.2.1
+      - laa-fee-calculator-client/1.1.0
```

Comment:
this might need rerecording against version 1.2 ?! or, as I do if possible, just change the cassette directly for small things like this if it causes failures

Reply:
👍 I have rerecorded the specs against v1.2.